### PR TITLE
Make range selector bigger

### PR DESCRIPTION
**Pull request by qingweibinary**
@borisyankov let me know the spec ;)
Now I simply make it bigger, do you want to position it lower? As it seems close already

screen shot 2016-06-02 at 2 18 19 pm

### CODE REVIEW

**Comment by qingweibinary** [comment]
Closer like this?

screen shot 2016-06-02 at 2 24 33 pm

**Comment by borisyankov** [comment]
Yeah, that looks pretty good ;)

File: .storybook/charts/ContractChart.js [hunk 1]
```diff
@@ -54,7 +54,15 @@ export default class ContractChart extends React.Component {
         const { ticks, contract } = this.state;
         const { contractId } = this.props;
         const getDataWhenChange = (count, type) =>
-            api.getDataForContract(() => getContract(contractId), count, type).then(newTicks => this.setState({ ticks: newTicks }));
+            api
+                .getDataForContract(() => getContract(contractId), count, type)
+                .then(newTicks => {
+                    if (contract) {
+                        this.setState({ ticks: newTicks, contract });
```
Comment (borisyankov) [inline .storybook/charts/ContractChart.js]: I wonder if this will not work too:
 this.setState({ ticks: newTicks, ...contract });

Comment (qingweibinary) [inline .storybook/charts/ContractChart.js]: It's storybook, I tried to simulate something closer to next-gen ;)

**Pull request merged**